### PR TITLE
Fix logout and show user info, Add remove from transfer

### PR DESCRIPTION
**Pull request by grdaneault**
Fixed #41 and made login look a little nicer (name actually shows up now)

### CODE REVIEW

File: src/main/webapp/js/controllers/TransferRequestController.js [hunk 1]
```diff
@@ -17,14 +18,10 @@ gyfp.controller("TransferRequestController", ["$scope", "$modal", "$routeParams"
...
     // Watch for change in authentication state
-    $scope.$watch(authService.isAuthenticated, function(isAuthenticated) {
+    $scope.$on("AuthenticationService.AuthenticationChanged", function(event, isAuthenticated) {
         $scope.authenticated = isAuthenticated;
         if (isAuthenticated) {
             $scope.loadRequest();
-        } else {
-            // If we are not authenticated, attempt to authenticate using
-            // existing credentials to prevent having to click the login button
-            authService.checkAuth();
```
Comment (regdoug) [inline src/main/webapp/js/controllers/TransferRequestController.js]: Just out of curiosity, why was this removed?

Comment (grdaneault) [inline src/main/webapp/js/controllers/TransferRequestController.js]: It was essentially causing an infinite loop, because checkAuth would cause
another failed auth broadcast, which would in turn cause another checkAuth,
causing ...

It needs to be reworked and added back at some point, but I'm not exactly
how/where yet.

Comment (regdoug) [inline src/main/webapp/js/controllers/TransferRequestController.js]: Made an issue so we don't forget to put it back in.
